### PR TITLE
Fix failing integration spec (including workaround)

### PR DESCRIPTION
**Pull request by unasuke**
## changes
- use system vagrant
    - install vagrant from rubygems is outdated https://www.vagrantup.com/docs/installation/
- use ubuntu 16.04 (xenial)
    - in ubuntu 14.04, package 'ruby' is EOL version (1.9)
-  update 'sl' version
- add 'not_if' attribute to execute mkdir resources
- remove 'action' attribute from  `file '/tmp/file_edit_with_suid' ` (workaround)
    - in `action: edit` causes exception `Net::SCP::Error`

## require
- use wercker ci classic strack
    - http://sue445.hatenablog.com/entry/2016/04/07/195627#Wercker%E3%81%AE%E8%A8%AD%E5%AE%9A
    - https://app.wercker.com/unasuke/itamae/runs 

## reference
- http://sue445.hatenablog.com/entry/2016/04/07/195627
- itamae plugins created by sue445

### CODE REVIEW

**Comment by unasuke** [comment]
@ryotarai Hou about it? Is something missing?

File: wercker.yml [hunk 1]
```diff
@@ -1,77 +1,57 @@
-box: wercker/rvm
-# Build definition
+box: drecom/ubuntu-ruby:2.3.1
+
 build:
```
Comment (ryotarai) [inline wercker.yml]: https://app.wercker.com/buildstep/5a6ef7bfc16a530100f6b298 says `build/steps is not of required type.`

```
build:
  steps:
```

should this be removed? (I'm not sure whether this fixes the problem...)

Comment (unasuke) [reply wercker.yml]: Thank you for a review!

The pull-req uses wercker classic stack and configure workflow and pipeline on web UI.

e.g. 
screenshot from 2018-04-02 18-41-21

So, please configure your wercker project `ryotarai/ itamae` :pray: 

see also http://sue445.hatenablog.com/entry/2016/04/07/195627#Wercker%E3%81%AE%E8%A8%AD%E5%AE%9A

Comment (unasuke) [reply wercker.yml]: oh……

Comment (sue445) [reply wercker.yml]: > The pull-req uses wercker classic stack 

No. This PR already looks like a workflow style.

I think we should change to workflow stack on console.

c.f. https://sue445.hatenablog.com/entry/2016/08/12/133645

**Comment by ryotarai** [comment]
Thank you for fixing this problem but https://github.com/itamae-kitchen/itamae/pull/258 looks better, I think. So let me close this PR. Thank you again.

**Pull request closed without merging**